sunpy/map/map_factory.py

```diff
@@ -120,15 +119,19 @@ class MapFactory(BasicRegistrationFactory):
     """  # noqa
 
     def _read_file(self, fname, **kwargs):
-        """ Read in a file name and return the list of (data, meta) pairs in
-            that file. """
-
-        # File gets read here.  This needs to be generic enough to seamlessly
+        """
+        Read in a file name and return the list of (data, meta) pairs in that file.
+        """
+        # File gets read here. This needs to be generic enough to seamlessly
         # call a fits file or a jpeg2k file, etc
         # NOTE: use os.fspath so that fname can be either a str or pathlib.Path
         # This can be removed once read_file supports pathlib.Path
         log.debug(f'Reading {fname}')
-        pairs = read_file(os.fspath(fname), **kwargs)
+        try:
+            pairs = read_file(os.fspath(fname), **kwargs)
+        except Exception as e:
+            msg = f'Failed to read {fname} because: {str(e)}'
```

Comment:
If we do the exception chaining below, we don't need this.
```suggestion
            msg = f"Failed to read {fname}."
```

Reply:
Fixed.